gen/definitions/system.yaml

```diff
@@ -134,6 +134,64 @@ attributes:
       - yang_name: server-ip-list-ordered
         tf_name: servers
         example: 2.3.4.5
+  - yang_name: ip/domain/lookup-settings/lookup/nsap
+    tf_name: ip_domain_lookup_nsap
+    example: true
+  - yang_name: ip/domain/lookup-settings/lookup/recursive
+    tf_name: ip_domain_lookup_recursive
+    example: true
+  - yang_name: ip/domain/lookup-settings/lookup/vrf
+    tf_name: ip_domain_lookup_vrfs
+    type: List
+    attributes:
+      - yang_name: vrf-name
+        tf_name: vrf
+        example: VRF1
+        id: true
+      - yang_name: source-interface/interface-choice/Loopback/Loopback
+        xpath: source-interface/Loopback
+        tf_name: source_interface_loopback
```

Comment:
I believe "tf_name" can be removed, as it would be derived (snake case) from "xpath" by default.